src/lib.rs

```diff
@@ -71,22 +71,14 @@
 
 #![deny(missing_docs)]
 #![cfg_attr(feature = "bench", feature(test))]
+#![feature(specialization)]
+#![feature(portable_simd)]
+#![feature(stdsimd)]
```

Comment:
These features turn this lib into a nightly-only lib, don't they? If so, that should be avoided.

Reply:
packed_simd and std::simd both need nightly... I wonder if we can detect nightly and turn it on or off accordingly